packages/loopring_v3/contracts/aux/bridge/IBridge.sol

```diff
@@ -42,8 +42,8 @@ interface IBridge
     ///      The sender will send the funds to Loopring exchange, so just like with normal
     ///      deposits the sender first has to approve token transfers on the deposit contract.
     ///
-    /// @param deposits The deposits
-    function batchDeposit(BridgeTransfer[] calldata deposits)
+    /// @param transfers The L2 transfers from Bridge to owners
+    function batchDeposit(BridgeTransfer[] calldata transfers)
```

Comment:
Isn't it a bit confusing to call these `transfers`, especially on the interface level with a function external people may want to use? People calling this function want to batch deposits, so the input is a bunch of `deposits`, that makes sense to me. The contract will handle them as efficiently as possible, but how this is done they don't have to worry to much about if they don't care.

But I can imagine having an input called `transfers` is confusing. And the new `param` description doesn't really help because the caller needs to specify a bunch of transfers from the bridge to the owners for some reason? Even when reading the complete description people wouldn't necessarily understand how this works because we would need to give more info in the description for this to make sense. But I don't see a reason to do so because people just want to do bunch of deposits together cheaper, no need to explain how it works exactly (which could always change).

That's why I still like to call them deposits, even internally sometimes. They are deposits, but the optimized implementation is to use L2 transfers but that's still just an implementation detail.

Reply:
I see your point, but the type is called BridgeTransfer but the variable name is called 'deposits', this is not consistent and is also confusing. So we have two options:
```
function batchDeposit(BridgeDeposit[] calldata deposits)
```

```
function batchDeposit(BridgeTransfer[] calldata transfers)
```

Which one is your pick？

Reply:
`function batchDeposit(BridgeDeposit[] calldata deposits)`!